android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -840,7 +840,7 @@ abstract class AbstractFetchDownloadService : Service() {
         intent.putExtra(EXTRA_DOWNLOAD_STATUS, getDownloadJobStatus(downloadState))
         intent.putExtra(EXTRA_DOWNLOAD_ID, downloadState.state.id)
 
-        context.sendBroadcast(intent)
+        context.sendBroadcast(intent, "org.mozilla.permission.RECEIVE_BROADCAST")
```

Comment:
Consider making this a public val or something that lets you add a kdoc that this is needed to be added to an application's manifest.

Reply:
@mcarare your latest patch looks good to me. Thanks!